src/frontend/src/sass/_collapsible.scss

```diff
@@ -7,18 +7,24 @@
     background-color: $bg-white;
     position: relative;
 
+    // This allows the background color to extend to the edge of the browser window
     &::before {
       content: '';
       height: 100%;
-      width: 100%;
+      width: 120%;
```

Comment:
120% ⭕ 🤔

Reply:
Yep, this is intended. Also note the `right: -120%`.